src/applications/mhv-medical-records/components/RecordList/ConditionListItem.jsx

```diff
@@ -28,7 +28,9 @@ const ConditionListItem = props => {
         {record.name}
       </span>
 
-      <p className="vads-u-margin--0">Date entered: {record?.date}</p>
+      <p className="vads-u-margin--0" data-dd-privacy="mask">
+        Date entered: {record?.date}
+      </p>
```

Comment:
In a case like this where we have a header next to some data, you can wrap the data in a `span` tag. So, something like `Date entered: <span data-dd-privacy="mask">{record?.date}</span>`